FirebaseAppDistribution/Sources/Private/FIRAppDistributionUIService.h

```diff
@@ -19,31 +19,26 @@
 #import "FIRAppDistribution+Private.h"
 
 NS_ASSUME_NONNULL_BEGIN
-
+/// An instance of this class provides UI services required for the App Distribution sdk
 /// An instance of this class is meant to be registered as an AppDelegate interceptor, and
```

Comment:
Should we remove or combine the previous comment?

Reply:
That was my intention. But I can make the comments have a better flow.

Reply:
I think just removing the duplicate `An instance` might do it - it looked like an oversight because of the duplicate structure. I see what you mean here - maybe `An instance of this class provides UI elements required for the App Distribution tester authentication flow as an AppDelegate interceptor.`